### PR TITLE
Create change log entry for release 3.0.1

### PR DESCRIPTION
**Pull request by stephenashank**
If documentation and test changes should be removed, let me know

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,26 @@
...
+### Added
+- Code formatting tools for pom.xml and java source files
+- Support for use with Configuration as Code plugin, with test dependencies on configuration-as-code
+  version 1.9
+- Secret used during integration test can be provided through GOOGLE_CREDENTIALS_FILE environment
+  variable which should be a path to the secret file
+- Documentation transferred from wiki.jenkins.io and stored in source
+
+### Changed
+- Use unique IDs for each instance
```
Comment (rachely3n) [inline CHANGELOG.md]: Can we change this to unique ID's for each cloud. The commit does say unique ID for each instance, but I think it was more meant to be unique instance ID for each cloud.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,26 @@
...
+- Test classes run in parallel using the concurrency property in pom.xml
```
Comment (rachely3n) [inline CHANGELOG.md]: Was this test classes in general or more specifically integration tests?

File: CHANGELOG.md (hunk 1 above)
Comment (stephenashank) [reply CHANGELOG.md]: Good catch, I'll update it.

File: CHANGELOG.md (hunk 2 above)
Comment (stephenashank) [reply CHANGELOG.md]: It actually applies to the unit tests as well. it probably accounts for maybe 30 seconds of speedup but still good to have.

File: CHANGELOG.md (hunk 1 above)
Comment (stephenashank) [reply CHANGELOG.md]: Done.

**Pull request merged**